bloopgun/src/main/scala/bloop/bloopgun/Bloopgun.scala

```diff
@@ -483,19 +501,26 @@ class BloopgunCli(
       }
 
       process.redirectErrorStream()
-      val started = process.start()
-      val is = started.getInputStream()
-      val code = started.waitFor()
-      val output = scala.io.Source.fromInputStream(is).mkString
-      StatusCommand(code, output)
+      try {
+        val started = process.start()
+        val is = started.getInputStream()
+        val code = started.waitFor()
+        val output = scala.io.Source.fromInputStream(is).mkString
+        StatusCommand(code, output)
+      } catch {
+        case e: IOException =>
+          StatusCommand(1, e.getMessage())
```

Comment:
We might want to give some context to this message, I imagine that if the error message is too generic we might have a hard time tracking where the error originated from. What do you think?